package.json

```diff
@@ -23,6 +23,7 @@
     "test.watch": "stencil test --spec --watch",
     "test.e2e": "stencil test --e2e",
     "release": "standard-version",
+    "github-release": "release-it",
```

Comment:
Isn't `release-it` alternative to `standard-version`? In this case, I would prefer to replace the `release` command instead of creating a new one.

Reply:
see above